src/runtime/internal/routes/openapi.ts

```diff
@@ -1,5 +1,6 @@
 import { type HTTPMethod, eventHandler, getRequestURL } from "h3";
 import type {
+  Extensable,
```

Comment:
```suggestion
  Extensible,
```

(+subsequent occurrences)

Reply:
https://github.com/nitrojs/nitro/blob/2145be0ed819531e64767ef4b1da8545d51f284d/src/types/openapi-ts.ts#L31-L33

It's named `Extensable` actually 😅